puppet/zulip/manifests/postgres_common.pp

```diff
@@ -43,6 +46,17 @@
       exec {'pip3_deps':
         command => 'python3 -m pip install pytz python-dateutil'
       }
+      group { 'ssl-cert':
+        ensure => present,
+      }
+      # allows ssl-cert group to read /etc/pki/tls/private
+      file { '/etc/pki/tls/private':
+        ensure => 'directory',
+        mode   => '0640',
+        owner  => 'root',
+        group  => 'ssl-cert',
+      }
+      $postgres_user_reqs = Package[$postgresql]
```

Comment:
This list should include `Group['ssl-cert']`.

Reply:
Fixed.